app/main.py

```diff
@@ -1,8 +1,60 @@
+from typing import Union
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: Union[float, int],
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def can_we_wash_car(self, car: Car) -> bool:
+        """
+        returns True if station.clean_power > car.clean mark else False
+        """
+        return self.clean_power > car.clean_mark
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
I think it's better to give more descriptive type annotation inside list

Reply:
Since it won't afford in any way I disagree. Yup, making Generics is more descriptive, but, really, according to README.md, we only producing list of Car's and if we make this annotation it won't help, till we get logs, so it's just excessive
![image](https://user-images.githubusercontent.com/107141441/216819994-11ddedc5-0ca3-4059-8349-c77078eb7382.png)